deps.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Instead of script, I would add a single Makefile with at least two targets:
`deps` that will install dependencies and `clean` that will clean up a directory. `make` is more standard way to do such job than custom scripts.

Reply:
here you go